pom.xml

```diff
@@ -1650,6 +1652,38 @@
 
         <pluginManagement>
             <plugins>
+                <plugin>
+                    <groupId>io.paradoxical</groupId>
+                    <artifactId>resolved-pom-maven-plugin</artifactId>
+                    <version>1.0</version>
+                    <configuration>
+                        <properties>
+                            <revision>${revision}</revision>
+                            <upstream.version>${upstream.version}</upstream.version>
+
+                            <!-- ADD HERE ALL THE CUSTOM PROPERTIES declared in the above <properties> part -->
+                            <!-- something like :
+                            <my.custom.property>${my.custom.property}</my.custom.property>
+                            -->
+                            <standard.features.tesb.version>${standard.features.tesb.version}</standard.features.tesb.version>
+                            <framework.features.tesb.version>${framework.features.tesb.version}</framework.features.tesb.version>
+                            <org.apache.aries.blueprint.core.tesb.version>${org.apache.aries.blueprint.core.tesb.version}</org.apache.aries.blueprint.core.tesb.version>
+                            <jline.tesb.version>${jline.tesb.version}</jline.tesb.version>
```

Comment:
should the pax-logging-log4j2.tesb.version also be added here?

Reply:
Yes, right!